examples/ucp_client_server.c

```diff
@@ -221,20 +311,30 @@ static ucs_status_t start_client(ucp_worker_h ucp_worker, const char *ip,
  * Print the received message on the server side or the sent data on the client
  * side.
  */
-static void print_result(int is_server, char *msg_str, int current_iter)
+static void print_result(int is_server, char **msg_str, int current_iter)
```

Comment:
can pass iov* instead of "char **msg_str", this way no need the copy_buffer() function

Reply:
@yosefe Wait for your feedback https://github.com/openucx/ucx/pull/6838/files#r672737268